packages/availity-workflow/scripts/lint.js

```diff
@@ -32,14 +34,29 @@ function lint() {
     spinner.color = 'yellow';
     spinner.start();
 
-    const files = settings.js();
+    // files tracked by git
+    let gitTrackedFiles;
+    const gitRootCmd = spawnSync('git', ['rev-parse', '--show-toplevel']);
+    if (gitRootCmd.status) {
+      // non-zero exit code; assume git repository absent
+      gitTrackedFiles = null;
+    } else {
+      const gitLsFiles = spawnSync('git', ['ls-files']);
+      gitTrackedFiles = gitLsFiles.stdout.toString().trim().split('\n');
```

Comment:
@robmcguinness I can combine lines 45 and 47, if you think it would make it more readable.